shaders/Crystal.gdshader

```diff
@@ -17,22 +17,24 @@ uniform vec4 emission : source_color = vec4(0.1f, 0.1f, 0.1f, 1.0f);
 uniform float emissionEnergy = 0.25f;
 
 uniform sampler2D dissolveTexture : source_color;
-uniform float dissolveValue : hint_range(0, 1) = 0.0f;
-uniform float dissolveScale : hint_range(0.5f, 5.0f, 0.1f) = 1.0f;
+uniform float dissolveValue : hint_range(0, 1) = 0.f;
+uniform float dissolveScale : hint_range(0.5f, 5.f, 0.1f) = 1.f;
 
-uniform float outlineWidth : hint_range(0.0f, 0.25f, 0.01f) = 0.0f;
-uniform vec4 growColor : source_color = vec4(0.1f, 0.1f, 0.1f, 1.0f);
+uniform float outlineWidth : hint_range(0.f, 0.25f, 0.01f) = 0.f;
+uniform vec4 growColor : source_color = vec4(0.1f, 0.1f, 0.1f, 1.f);
```

Comment:
These changes make the script less consistent with our Thrive coding style (C# doesn't support the 0.f syntax, instead it requires 0.0f, so we cannot use it in the codebase in general)...

I'll do a quick revert on this part as long as I don't find anything I'd need to ask you on how to fix in this PR.